forest/daemon/src/daemon.rs

```diff
@@ -87,13 +85,21 @@ pub(super) async fn start(opts: CliOpts, config: Config) -> anyhow::Result<Db> {
     let net_keypair = match get_keypair(&path.join("keypair")) {
         Some(keypair) => Ok::<forest_libp2p::Keypair, std::io::Error>(keypair),
         None => {
-            let gen_keypair = ed25519::Keypair::generate();
+            let gen_keypair = forest_libp2p::Keypair::generate_ed25519();
```

Comment:
Sigh, it's silly that they don't have a way to convert `ed25519::Keypair` into ` Keypair`. Oh well.

Reply:
Could you open an issue with libp2p and describe our use-case?

Reply:
https://github.com/libp2p/rust-libp2p/issues/3618